.circleci/config.yml

```diff
@@ -78,7 +78,7 @@ commands:
             cd << parameters.dir >> && npm ci && cd $ROOT_DIR
       # Save the cache to avoid extraneous downloads
       - save_cache:
-          key: node-v1-<< parameters.dir >>-{{ checksum "package-lock.json" }}
+          key: node-v3-<< parameters.dir >>-{{ checksum "package-lock.json" }}
```

Comment:
checking if regenerating checksum got rid of the test failure issue in ci?

Reply:
no, but it helped fixed the split node 14 / node 16 state.